packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -2283,7 +2271,7 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
 
     // Cache camera props so we can trigger one camera changed event after
     // The full transition.
-    const previousCameraProps = cloneDeep(this.getCamera());
+    const previousCameraProps = structuredClone(this.getCamera());
```

Comment:
Doesn't getCamera return a new camera instance every time?  If so, the structured clone isn't necessary.